app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
It feels a bit strange to have the list author and date only separated by a space. I'd suggest to include the date in the "created by" string and have it separated by a comma: `created by {listAuthor}, {date}`.